src/lib/api.js

```diff
@@ -2,6 +2,35 @@ const request = require('request-promise-native');
 
 const archivingApi = require('./archiving-api');
 const environment = require('./environment');
+const log = require('../lib/log');
+
+const logDeprecatedTransitions = (settings) => {
+  const uri = `${environment.instanceUrl}/api/v1/settings/deprecated-transitions`;
+
+  return request({ uri, method: 'GET', json: true})
+    .then(transitions => {
+      const appSettings = JSON.parse(settings);
+
+      if (appSettings.transitions) {
+        (transitions || []).forEach(transition => {
+          const transitionSetting = appSettings.transitions[transition.name];
+
+          if (transitionSetting === true || (transitionSetting && transitionSetting.disable === false)) {
```

Comment:
This is pretty hard to read (redundant) and I'm not sure it's correct. 
This is how we check for transitions being enabled in cht-core (here https://github.com/medic/cht-core/blob/master/shared-libs/transitions/src/transitions/index.js#L140):
```
 const disabled = conf && conf.disable;
 if (!conf || disabled) {
```

This means that if `conf === {}` (an empty object), the transition is enabled in `cht-core`. In your code, you require the `disabled` property to be explicitly `false` in order to consider it being false.

Reply:
Yes, I agree that it's not nice to read 😅... but if I follow what we do in the transitions/index.js

  const disabled = transitionSetting && transitionSetting.disable;
  if (transitionSetting && !disabled) {
         print message
  }
Then it will print the message when 
` "update_sent_forms": {
      "disable": false
    },`

Then, I can do following as middle ground (?) 🙂
  const disabled = transitionSetting && transitionSetting.disable;
  if (transitionSetting === true && !disabled) {
         print message
  }

Reply:
Done! :)